ui_framework/src/components/tabs/_tabs.scss

```diff
@@ -0,0 +1,52 @@
+.kuiTabs {
```

Comment:
Also, you'll notice I did a subtraction to get that 12px value again to pad out the tabs a bit. I guess we should make that `$kuiSizeM` that value, heh.